wiring/inc/spark_wiring_system_power.h

```diff
@@ -105,6 +107,20 @@ class SystemPowerConfiguration {
         return conf_.soc_bits;
     }
 
+    SystemPowerConfiguration& auxPowerControlPin(uint8_t pin, bool activeLevel = 1) {
```

Comment:
Nit: I'd use more verbose names here e.g. auxiliaryPowerControlPin() just for readability.